exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExportUtils.java

```diff
@@ -151,8 +150,7 @@ static MetricKind createMetricKind(AggregationWindow window) {
     // Shared fields for all TimeSeries generated from the same ViewData
     TimeSeries.Builder shared = TimeSeries.newBuilder();
     shared.setMetricKind(createMetricKind(view.getWindow()));
-    // TODO(songya): add support for custom resource labels.
-    shared.setResource(MonitoredResource.newBuilder().setType("global"));
+    shared.setResource(StackdriverStatsMonitoredResource.getMonitoredResource());
```

Comment:
Suggestion: This method would be clearer and easier to test if it took the monitored resource as an argument instead of accessing a static variable.

Reply:
Done.